### ISSUE TITLE
organize examples

### ISSUE THREAD

**Issue by rbrugaro** [opea-project/GenAIExamples#896]
Issue: there are many examples and is getting harder to know what they are about just from the example name

My recommendation would be to:
1)group examples into bucket folders
2)add a left column in the examples table indicating bucket and sort the rows

![image](https://github.com/user-attachments/assets/f9eeb563-c5e2-47cc-a53f-a044a2915094)

**Comment by mkbhanda** [comment]
Agree, nice to bucket by some commonality. Your suggestion needs some tweaks .. the RAG apps also use LLM. Perhaps instead of Model-lifecycle just "finetuning"? Our naming convention could improve too, only GraphRAG has RAG in its name.

**Comment by rbrugaro** [comment]
@mkbhanda yes we can find better names. My reasoning is the G in RAG implies llm generation and highlights retrieval of some sort is used.. we could do retrieval based and non retrieval based... 
Model lifecycle can be broader than just finetuning for example if we want broader examples including trace logging and evaluations together with finetune (common in deployments)... or examples with model_guard+llm or prompt shields+llm.. those could fit in this bucket I would think.

**Comment by yinghu5** [comment]
Hi @ftian1 @mkbhanda  just remind may consider it in the further release, thank you!
